torch/nn/quantized/functional.py

```diff
@@ -5,17 +5,61 @@
 from __future__ import unicode_literals
 
 import torch
-from torch._ops import ops
-from torch._jit_internal import List
+from torch._ops import ops as _ops
+from torch._jit_internal import List as _List
 
 def _extend_to_list(val, length=2):
     if not isinstance(val, (tuple, list)):
         val = [val] * length
     return val
 
-relu = ops.quantized.relu
-add_relu = ops.quantized.add_relu
+def relu(input, inplace=False):
+    # type: (Tensor, bool) -> Tensor
+    r"""relu(input, inplace=False) -> Tensor
 
+    .. note::
+      :attr:`inplace` is not supported for the quantized relu.
+
+    Applies the rectified linear unit function element-wise. See
+    :class:`~torch.nn.quantized.ReLU` for more details.
```

Comment:
again, `torch.nn.functional.ReLU` is what you meant to type here

Reply:
Actually, about that -- don't we want to refer the functional to its module counterpart? The documentation in the `torch.nn.quantized.ReLU` is more relevant to this than the non-quantized counterpart